### PR TITLE
Saving files and data to tempdir

### PR DESCRIPTION
**Pull request by heemanshusuri**

### CODE REVIEW

File: tests/testthat/test_datastore.R [hunk 1]
```diff
@@ -10,21 +10,21 @@ test_that("default datastore", {
   upload_files_to_datastore(ds, files = list(file.path(".", file_name)))
 
   # download files from datastore
-  target_dir <- "./downloaded_files"
+  target_dir <- file.path(tempdir(),"downloaded_files")
```
Comment (ninghu) [inline tests/testthat/test_datastore.R]: missing space after comma

Comment (ninghu) [reply tests/testthat/test_datastore.R]: same problem for other places as well.

**Comment by ninghu** [comment]
Samples are all good?

File: tests/testthat/test_datastore.R (hunk 1 above)
Comment (heemanshusuri) [reply tests/testthat/test_datastore.R]: resolved

**Pull request merged**